dll/win32/shell32/folders/CPrinterFolder.cpp

```diff
@@ -436,7 +436,7 @@ HRESULT WINAPI CPrinterFolder::GetDefaultColumnState(UINT iColumn, DWORD *pcsFla
 {
```

Comment:
Change the type of pcsFlags to SHCOLSTATEF* ?

Reply:
Changed. Thanks.